server/src/main/java/io/crate/types/TypeSignature.java

```diff
@@ -233,6 +238,10 @@ public String toString() {
         return typeName.toString();
     }
 
+    public boolean equalsIgnoringParameters(TypeSignature o) {
```

Comment:
only a suggestion to shorten the name (also fine as is):
```suggestion
    public boolean equalsIgnoreParams(TypeSignature o) {
```